.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,42 @@
+name: "\U0001F41E Bug report"
+description: Report an issue with Hilla
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is. If you intend to submit a PR for this issue later, please let us know here.
+      placeholder: Bug description
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected-behavior
+      description: Describe what the expected behavior you think, and we will validate on our designs. 
+      placeholder: expected behavior
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction
+      description: Please tell us how to reproduce this issue step by step. If you have an example project to show, please provide us the link. 
+      placeholder: Reproduction
+    validations:
+      required: true
+  - type: textarea
+    id: system-info
+    attributes:
+      label: System Info
+      description: Please let us know your environment when reproducing the issue. 
+      placeholder: Operating System, Hilla version, Browsers type and version
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        If you think this ticket is important, add a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to help the community and maintainers prioritize this issue.

```

Comment:
Doesn't everyone think their tickets are important, especially when they put some time and effort to create a ticket? :) I think I understand the intention behind this, but probably we should reword it to communicate what we mean about the prioritization, or possibly to remove this from the Bug Report template.

Reply:
> Doesn't everyone think their tickets are important

This message is not for the author of the ticket. It's for the people reading it. It is the same phrasing used by the docs: https://github.com/vaadin/docs/blob/latest/.github/ISSUE_TEMPLATE/doc_report.md

Reply:
Another formulation could be the one used by this [project](https://github.com/aws/containers-roadmap/blob/master/.github/ISSUE_TEMPLATE/community-request-.md):

_Please vote on this issue by adding a 👍 reaction to the original issue to help the community and maintainers prioritize this request_

Reply:
I think this suggestion is way better than the one from docs. I believe it could be used on both issue and feature request templates.